build/includes/sdk.mk

```
@@ -213,3 +213,21 @@ sdk-shell-csharp:
 sdk-publish-csharp: RELEASE_VERSION ?= $(base_version)
 sdk-publish-csharp:
 	$(MAKE) run-sdk-command-csharp COMMAND=publish VERSION=$(RELEASE_VERSION) DOCKER_RUN_ARGS="$(DOCKER_RUN_ARGS) -it"
+
+# difference in sdks before and after gen-all-sdk-grpc target
+test-gen-all-sdk-grpc:
+	mkdir -p includes/tmp
+	cp -r ../sdks includes/tmp
+	cp -r ../test/sdk includes/tmp
+	make gen-all-sdk-grpc
+	diff_output=$$(diff -bBr includes/tmp/sdks ../sdks); \
+	diff_output_test_sdk=$$(diff -bBr includes/tmp/sdk ../test/sdk); \
+	if [ -z "$$diff_output" ] && [ -z "$$diff_output_test_sdk" ]; then \
```

```suggestion
	@if [ -z "$$diff_output" ] && [ -z "$$diff_output_test_sdk" ]; then \
```

I think should work nicely to hide the if statement itself from being printed, so we only see the output.